src/test/scala/org/broadinstitute/dsde/workbench/sam/service/UserServiceSpecs/SupportSummarySpec.scala

```diff
@@ -0,0 +1,208 @@
+package org.broadinstitute.dsde.workbench.sam.service.UserServiceSpecs
```

Comment:
These are the tests moved from `UserRoutesV2Spec`, but I also cleaned them up a bunch to avoid code repetition and Sonar warnings